docs/examples/sierpinski.html

```diff
@@ -0,0 +1,158 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta http-equiv=X-UA-Compatible content=IE=edge>
+    <meta name=viewport content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <title>DIO - Sierpinski Triangle</title>
+    <link rel=icon type=image/png href=../imgs/favicon-large.png sizes=32x32>
+    <link rel=icon type=image/png href=../imgs/favicon-small.png sizes=16x16>
+    <script src=https://unpkg.com/dio.js@latest/dist/dio.min.js></script>
+  </head>
+  <body>
+    <div id="root"></div>
+
+    <style>
+      html {
+        width: 100%; 
+        height: 100%; 
+        overflow: hidden
+      }
+    </style>
+
+    <script>
+      var dotStyle = {
+        position: 'absolute',
+        background: '#61dafb',
+        font: 'normal 15px sans-serif',
+        textAlign: 'center',
+        cursor: 'pointer',
+      };
+
+      var containerStyle = {
+        position: 'absolute',
+        transformOrigin: '0 0',
+        left: '50%',
+        top: '50%',
+        width: '10px',
+        height: '10px',
+        background: '#eee',
+      };
+
+      var targetSize = 25;
+
+      class Dot extends dio.Component {
+        constructor() {
+          super();
+          this.state = { hover: false };
+        }
+        enter() {
+          this.setState({
+            hover: true
+          });
+        }
+        leave() {
+          this.setState({
+            hover: false
+          });
+        }
+        render() {
+          var props = this.props;
+          var s = props.size * 1.3;
+          var style = Object.assign({}, dotStyle, {
+            width: s + 'px',
+            height: s + 'px',
+            left: (props.x) + 'px',
+            top: (props.y) + 'px',
+            borderRadius: (s / 2) + 'px',
+            lineHeight: (s) + 'px',
+            background: this.state.hover ? '#ff0' : dotStyle.background
+          });
+          return (
+            h('div', {style: style, onMouseEnter: ()=>this.enter(), onMouseLeave: ()=>this.leave()},
+              this.state.hover ? '*' + props.text + '*' : props.text
+            )
+          );
+        }
+      }
+
+      function SierpinskiTriangle({ x, y, s, children }) {
+        if (s <= targetSize) {
+          return (
+            h(Dot, {x: x - (targetSize / 2), y: y - (targetSize / 2), size: targetSize, text: children})
+          );
+          return r;
+        }
+        var newSize = s / 2;
+        var slowDown = false;
```

Comment:
is this a debug prop? never `true`

Reply:
Yes, it artificially creates a long running work load to test cooperative scheduling when it lands in the next minor release. Everything about the implementation is not optimized for synchronous performance to test that end.